### DISCUSSION TITLE
Suddenly won't boot

### DISCUSSION THREAD

**Discussion by otabwe** [rern/rAudio#1147] [Bug Reports]
Greetings! I have been using RuneAudio and then rAudio for years on Rasp Pi 3B+ with a HiFi Berry Digi+. Quite suddenly, about a week ago, everything stopped working. I have downloaded and reflashed to v20221117 several times using a couple of methods. rAudio doesn't get past the opening logo screen. I can't find the device using terminal, etc. In the event it was the device malfunctioning, I flashed Raspian, which booted up just fine. I am completely clueless as to the cause and sad I cannot listen to my music... Any insight/assistance would be most appreciated. Thanks, Otabwe

**Comment by klassikmann** [comment]
@otabwe 
I'm using almost the same configuration: RPi 3B+ with HiFiBerry Digi+ Pro. I seem to remember a similar situation some time ago. My music is provided by a CIFS network share. Once the component was unreachable, it looked like rAudio couldn't boot.
An idea only.

**Reply by otabwe** [reply]
Greetings klassikmann!
Thank you for the prompt reply! My music is on usb drives attached directly to the pi, so I am doubtful that there's a problem having to do with the network? Perhaps I am missing something...
Warm wishes for the New Year!
Otabwe

**Comment by voterpublic** [comment]
Does it boot if you remove the HiFi Berry Digi+?
I had a similar issue where I could not get it working & thought my HiFi Berry DAC+ was dead. I asked for help from HiFi Berry. They sent me their software to try. It worked thus proving there was nothing wrong with my HiFi Berry.
I did a fresh install on a fully formatted SD card and it worked. Been over a year & it has not happened again. Good luck.

**Reply by otabwe** [reply]
Greetings voterpublic!
I just removed the Hifi Berry digi+ and rebooted. Same result. I still only see the rAudio logo stuck in the middle of the screen.
I am going to try yet again to scrub the SD card and reflash, reboot with the Hifi Berry still off. Cross my fingers and such...
Thank you for your suggestion!
Warm wishes for the New Year!
Otabwe

**Reply by Lightwel** [reply]
This happened to me on RPi 4B.

On boot screen (just rAudio logo displayed)

Ctrl-Alt-F2

login > root
pw > ros

rAudio : startx

Wait.

If nothing appears to happen, wait. 

If still nothing appears to happen, place the mouse on the logo and move it around.  If the logo is 'erased' then move the mouse around the screen to see if it is revealed. 

Then set up as normal. 

I do no know how to fix this permanently, I have to do it on every boot.

Best
R
